content/_includes/schedule.njk

```diff
@@ -2,7 +2,15 @@
 {% set speaker = speakers | filterByKey('name', speakerName) | first %}
 <div class="speaker sm:flex items-start flex-row-reverse">
   <div class="sm:w-1/2 xl:w-3/4 mb-4 sm:mb-0 sm:pl-6">
-    <h3 class="font-bold text-xl sm:text-3xl !mt-0 italic">{{ speaker.talkTitle | noRunts | safe }}</h3>
+    <h3 class="font-bold text-xl sm:text-3xl !mt-0 italic">
+      {{ speaker.talkTitle | noRunts | safe }}
+    </h3>
+    <div>
+      <a class="button primary ; !normal-case !p-2 mr-2 mb-2" href="{{ speaker.youtubeWatch }}" target="_blank">
+        <span class="inline-block align-middle"><i class="ri-youtube-fill ri-2x"></i></span>
+        <span class="inline-block align-middle">Watch Now!</span>
+      </a>
```

Comment:
Can we make the text white on these orange buttons? (Specifically, to have the button color match the background – so if on white bg, white text or purple bg, purple text)